.gitpod.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gitpod/workspace-full@aad6c2650705
+FROM gitpod/workspace-full
```

Comment:
Codacy found an issue: [Always tag the version of an image explicitly](https://app.codacy.com/manual/lucaslarson/HQ9/pullRequest?prid=6295927)

Reply:
> Codacy found an issue: [Always tag the version of an image explicitly](https://app.codacy.com/manual/lucaslarson/HQ9/pullRequest?prid=6295927)

@Codacy must know how to implement Dockerfile versioning if this wasn’t the way (Gitpod fails to build with the tag `@aad6c2650705`)

Reply:
a correct way to specify versioned Dockerfile sources is the answer is by adding `@sha256:𝑥` where 𝑥 is a 64‑character hexadecimal string and the Docker digest number. This version number `aad6c2650705` was the first 12 characters of a digest string and had no `@sha256:` prefix.

https://github.com/gosh-terminal/gosh/commit/e5a40c2cb90a595b8181a4a3a053568a9b0a3785